crd/fleet/fleet-crds.yaml

```diff
@@ -1569,93 +1640,739 @@ spec:
-                  sidecarRef:
-                    description: SidecarRef is a reference to a sidecar Custom Resource.
-                      If set, sidecar containers included in the sidecar Custom Resource
-                      will be injected into the database pod.
-                    properties:
-                      name:
-                        description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
-                          TODO: Add other useful fields. apiVersion, kind, uid?'
-                        type: string
-                    type: object
-                    x-kubernetes-map-type: atomic
-                  sourceCidrRanges:
-                    description: Source IP CIDR ranges allowed for a client.
-                    items:
-                      type: string
-                    type: array
-                  tls:
-                    description: TLS is the desired server certificate configuration
-                      for the instance. This field is optional. When this field is
-                      changed, the instance pods will restart to load the specified
-                      certificate
-                    properties:
-                      certSecret:
-                        description: CertSecret contains the name of a certificate
-                          secret within the same namespace. The secret must contain
-                          entries ca.crt (CA certificate), tls.key (server private
-                          key), and tls.crt (server leaf certificate). This secret
-                          is used to set the TLS config for the database instance.
+                      podAffinity:
+                        description: PodAffinity describes pod affinity scheduling
+                          rules for the instance.
                         properties:
-                          name:
-                            description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
-                              TODO: Add other useful fields. apiVersion, kind, uid?'
-                            type: string
+                          preferredDuringSchedulingIgnoredDuringExecution:
+                            description: The scheduler will prefer to schedule pods
+                              to nodes that satisfy the affinity expressions specified
+                              by this field, but it may choose a node that violates
+                              one or more of the expressions. The node that is most
+                              preferred is the one with the greatest sum of weights,
+                              i.e. for each node that meets all of the scheduling
+                              requirements (resource request, requiredDuringScheduling
+                              affinity expressions, etc.), compute a sum by iterating
```

Comment:
the scheduler does these calculations and picks the node that has the highest sum of weights.
description starts with the line "The scheduler will prefer..."

Reply:
how about this

```
description: The scheduler will prioritize nodes that meet the affinity
                              expressions specified in this field when scheduling pods,
                              but it may still choose a node that doesn't fully meet the
                              criteria. The most preferred node is the one with the highest
                              total weight. This weight is calculated by adding up values
                              for each node that satisfies all scheduling requirements
                              (such as resource requests or requiredDuringScheduling affinity
                              expressions). For each matching podAffinityTerm, the node's weight
                              increases, and the node(s) with the highest total weight are
                              selected as the most preferred.
```

Reply:
Please replace "may" with "might" -- otherwise, LGTM! Thank you.